aws/eks/locals.tf

```diff
@@ -14,6 +14,7 @@ locals {
     "metrics_server"     = true
     "reloader"           = true
     "velero"             = true
+    "prometheus"         = false
```

Comment:
I'm not specifically opposed to this value for sanity but just to be clear, all of our clusters will have prometheus, right? If so then would we not want to have the default be true? If the idea here though is to start with this set to false for safety then change the default later I'm fine with that.

Reply:
Yes this is for safety sake, we should set this to false for now until we do the full roll out